clang/test/CIR/IR/annotations.cir

```diff
@@ -0,0 +1,20 @@
+// RUN: cir-opt %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
```

Comment:
Please also create another test file `invalid-annotations.cir` where you test the verifier code you added. See `invalid.cir` if you need examples.